src/main/java/io/vertx/core/metrics/impl/DummyVertxMetrics.java

```diff
@@ -82,7 +81,7 @@ public TCPMetrics createMetrics(SocketAddress localAddress, NetServerOptions opt
   }
 
   @Override
-  public TCPMetrics createMetrics(NetClient client, NetClientOptions options) {
```

Comment:
it's not a public API and it can be changed, nobody really implements that except us in Dropwizard and Hawkular